<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigureDialog</class>
 <widget class="QDialog" name="ConfigureDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>303</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configure Fieldwork Somso Muscle Step</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="configGroupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_identifier">
        <property name="text">
         <string>identifier:  </string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit_identifier"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_input_unit">
        <property name="text">
         <string>Input unit:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="comboBox_in_unit"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_output_unit">
        <property name="text">
         <string>Output unit:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="comboBox_out_unit"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_write_osim_file">
        <property name="text">
         <string>Write Osim file:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QCheckBox" name="checkBox_write_osim_file">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_update_knee_splines">
        <property name="text">
         <string>Update Knee Splines:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QCheckBox" name="checkBox_update_knee_splines">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_static_vas">
        <property name="text">
         <string>Static Vastus:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QCheckBox" name="checkBox_static_vas">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_update_max_iso_forces">
        <property name="text">
         <string>Update Max Iso Forces:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QCheckBox" name="checkBox_update_max_iso_forces">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_subject_height">
        <property name="text">
         <string>Subject Height (m):</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QLineEdit" name="lineEdit_subject_height"/>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="label_subject_mass">
        <property name="text">
         <string>Subject Mass (kg):</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QLineEdit" name="lineEdit_subject_mass"/>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="label_output_dir">
        <property name="text">
         <string>Output folder:</string>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLineEdit" name="lineEdit_osim_output_dir"/>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_osim_output_dir">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEdit_identifier</tabstop>
  <tabstop>lineEdit_subject_height</tabstop>
  <tabstop>lineEdit_subject_mass</tabstop>
  <tabstop>comboBox_in_unit</tabstop>
  <tabstop>comboBox_out_unit</tabstop>
  <tabstop>checkBox_write_osim_file</tabstop>
  <tabstop>checkBox_static_vas</tabstop>
  <tabstop>checkBox_update_max_iso_forces</tabstop>
  <tabstop>lineEdit_osim_output_dir</tabstop>
  <tabstop>pushButton_osim_output_dir</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ConfigureDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ConfigureDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
